### ISSUE TITLE
Content removal - Clothes

### ISSUE THREAD

**Issue by JamesTDG** [MinicraftPlus/minicraft-plus-revived#295]
**Is your feature request related to a problem? Please describe.**
With the advent of skins being added, different colors for clothes are pretty much useless.

**Describe the solution you'd like**
Remove clothes from the game. This may make colored wool obsolete, but we could replace colored wool with dyes, which can make new features for the game easier to be added, such as colorable blocks and items. Colorable blocks/items could be customized with a certain color hex or set of hexes in place that texture creators can use to allow this feature to exist or not in their packs.

**Comment by BenCheung0422** [comment]
We could also change the behaviour of clothes into armor-like.

**Comment by Makkkkus** [comment]
I agree with Ben here. While I get that clothes are a bit useless with the new skin system, they are dependent on a mechanic that is pretty neat, so not using it would be bad. If anyone has any ideas on how to use wool, they would be greatly appreciated.

**Comment by JamesTDG** [comment]
@Makkkkus well, we could make banners to keep the usage of wool, if we really want to keep clothes, have a specific set of colors that can change on your character's skin, ofc make it a set of colors nobody would want to commonly use on their skins, like bright magenta.

**Comment by Makkkkus** [comment]
Yeah, I agree with this.

**Comment by BenCheung0422** [comment]
This issue may be superseded by #508 as https://github.com/MinicraftPlus/minicraft-plus-revived/issues/508#issuecomment-2480693898.
